tests/test_bonded.py

```diff
@@ -128,55 +128,51 @@ def test_rmsd_restraint(self):
         n_particles_a = 25
         n_particles_b = 7
 
-        relative_tolerance_at_precision = {np.float32: 2e-5, np.float64: 1e-9}
+        for _ in range(100):
+            coords = self.get_random_coords(n_particles_a + n_particles_b, 3)
 
-        for precision, rtol in relative_tolerance_at_precision.items():
```

Comment:
This just made the test slower and the values weren't used.

Reply:
yes the PR from the rmsd_patch branch also addressed this. I'll probably need to resolve a merge conflict with the RMSD patch later though, but I think the performance looks good.